spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -40,7 +40,7 @@
       def do_post
         post :create, params: { 'user' => {
           'email' => 'foo@bar.com',
-          'password' => 'passw0rd'
+          'password' => 'mypassw0rd'
```

Comment:
What's the reason for the password change? Is the value irrelevant?

Reply:
It is a test, previously it passed with 8 chars as that was the minimum, now we've increased the minimum to 10 chars, so this test is also updated for that reason.

Reply:
Doh! I couldn't see the value being checked for and overlooked the obvious requirement!